libtcmu_log.h

```diff
@@ -32,6 +32,16 @@
 #define TCMU_LOG_INFO	LOG_INFO	/* informational */
 #define TCMU_LOG_DEBUG	LOG_DEBUG	/* debug-level messages */
 
+typedef enum {
+        TCMU_LOG_TO_STDOUT,
+        TCMU_LOG_TO_STDERR,
+        TCMU_LOG_TO_SYSLOG,
+        TCMU_LOG_TO_FILE,
+} tcmu_log_destination;
+
+typedef int (*log_output_func) (int priority, const char *timestamp, const char*str, void *data);
+typedef void (*log_close_func) (void *data);
```

Comment:
Instead of _func could you use _fn to match some other patches that will be merged soon.